client/src/components/Error/Error.css

```diff
@@ -0,0 +1,17 @@
+/* The idea here is: push the container to the exact middle of the 
+browser window with left: 50%;, then pull it back to the left edge 
+with negative -50vw margin. */
+.error-background {
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+    left: 50%;
+    right: 50%;
+    margin-left: -50vw;
+    margin-right: -50vw;
+    /* This scales the background nicely with different browsers */
+    background-image: url('../../assets/404-error-page.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
```

Comment:
See inline comments. This is what makes our image take up the whole screen and scale nicely.